Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -2105,10 +2106,10 @@ public override void Update(float elapsedClockSeconds)
                     if (DynamicBrakeCommandStartTime + DynamicBrakeDelayS < Simulator.ClockTime)
                     {
                         DynamicBrake = true; // Engage
-                        if (IsLeadLocomotive() && DynamicBrakeIntervention >= 0)
+                        if (IsLeadLocomotive() && DynamicBrakeController.CurrentValue > 0)
```

Comment:
I'm getting a null reference exception in trains with blended braking but no physical controller. You have to check if DynamicBrakeController is null

Reply:
I tested with MSTS's HPP-8

Reply:
Fixed now, anything that runs into a dynamic brake controller when it isn't assumed to be present is nullable/null-coalescing/null checked. I also found some situations where combined control was checking for dynamic brakes even when not present (like the MSTS series 7000) and those are better now.